pkg/cmd/resource/operation_test.go

```diff
@@ -38,7 +38,7 @@ func TestRunOperationCmd(t *testing.T) {
 		require.Equal(t, http.MethodPost, r.Method)
 		require.Equal(t, "/v1/bars/bar_123", r.URL.Path)
 		require.Equal(t, "Bearer sk_test_1234", r.Header.Get("Authorization"))
-		require.Equal(t, "param1=value1&param2=value2", string(body))
+		require.True(t, string(body) == "param1=value1&param2=value2" || string(body) == "param2=value2&param1=value1")
```

Comment:
because ordering in maps isn't guaranteed

Reply:
Oof, good point. We should deserialize the form encoded string using [`ParseQuery`](https://golang.org/pkg/net/url/#ParseQuery) instead and test the individual values. Would make it easier to test when there are more than two keys in the query string.